sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -263,6 +228,25 @@ class ConvertHelper : public ReturnHelper {
   }
 };
 
+// Handle mismatched PI and UR type return sizes for info queries
+inline pi_result fixupInfoValueTypes(size_t ParamValueSizeRetUR,

```

Comment:
This function does not do any validation/error handling, perhaps making it a `void` return would be better?